src/Locale/de/cake.po

```diff
@@ -246,7 +246,7 @@ msgstr "Start"
 
 #: View/Widget/DateTimeWidget.php:540
 msgid "January"
-msgstr "Januar"
+msgstr "Jänner"
```

Comment:
I really don't care - I always use "Jänner" for years - and some others use "Januar" for years. Both are correct. Revert it to "Januar" if you feel comfortable with it. I personally like "Jänner" a lot more but that's just me. I just wanted to help a bit.